OpenTap.Plugins.PNAX/Network Analyzer Steps/Preset.cs

```diff
@@ -1,5 +1,5 @@
-﻿// Author: MyName
-// Copyright:   Copyright 2023 Keysight Technologies
+﻿// Author: CMontes
+// Copyright:   Copyright 2024 Keysight Technologies
```

Comment:
Technically this should become **2023-2024**

Reply:
done